### PR TITLE
[CU-https://app.clickup.com/t/302tgzn] Add a building of composable-centauri docker image

### PR DESCRIPTION
**Pull request by andor0**
## Issue
https://app.clickup.com/t/302tgzn

### CODE REVIEW

File: scripts/polkadot-launch/rococo-local-bridge-dali-dev.nix [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        "--beefy"
+      ];
+    };
+    parachains = [
+      {
+        id = 2001;
+        port = 31200;
+        wsPort = 9988;
+        count = 3;
+        chain = "dali-2001";
```
Comment (dzmitry-lahoda) [inline scripts/polkadot-launch/rococo-local-bridge-dali-dev.nix:33]: would be simpler if one network will be 2087 as it was

**Review by dzmitry-lahoda — APPROVED** [review]

File: scripts/polkadot-launch/rococo-local-bridge-dali-dev.nix (hunk 1 above)
Comment (andor0) [reply scripts/polkadot-launch/rococo-local-bridge-dali-dev.nix:33]: The bridge team asked so.

**Comment by andor0** [comment]
This is duplicate of https://github.com/ComposableFi/composable/pull/1849.

File: scripts/polkadot-launch/rococo-local-bridge-dali-dev.nix (hunk 1 above)
Comment (dzmitry-lahoda) [reply scripts/polkadot-launch/rococo-local-bridge-dali-dev.nix:33]: i see. so -bridge- is infix in file name. while in it is prefix (good) in flake. so naming all bridge team with bridge- prefix could be good. 

pablo team cannot use 2001 as all tooling ant rest rely on 2087.

Comment (andor0) [reply scripts/polkadot-launch/rococo-local-bridge-dali-dev.nix:33]: > i see. so -bridge- is infix in file name. while in it is prefix (good) in flake. so naming all bridge team with bridge- prefix could be good.

fixed in https://github.com/ComposableFi/composable/pull/1849

> pablo team cannot use 2001 as all tooling ant rest rely on 2087.

perhaps, we will change para_id to 2087 in the future.

**Pull request closed without merging**